src/main/java/com/customerApi/api/controllers/CustomerController.java

```diff
@@ -0,0 +1,64 @@
+package com.customerApi.api.controllers;
+
+import java.util.List;
+import java.util.Optional;
+
+import javax.validation.Valid;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.customerApi.api.modals.Customer;
+import com.customerApi.api.services.CustomerService;
+
+@RestController
+@RequestMapping("/customer")
+public class CustomerController {
+
+	@Autowired
+	private CustomerService service;
+
+	@PostMapping("/")
+	public ResponseEntity<Customer> saveCustomer(@Valid @RequestBody Customer data) {
+		Customer customer = service.addCustomerData(data);
+		return new ResponseEntity<>(customer, HttpStatus.OK);
```

Comment:
Can you convert this into single line?

Reply:
Thanks for your feedback. It's been resolved.